LiveSplit.AutoSplitters.xml

```diff
@@ -23190,3 +23190,12 @@
         <Description>Autosplitter</Description>
     </AutoSplitter>
 </AutoSplitters>
+<Games>
+    <Game> Disney Epic Mickey: Rebrushed</Game>
+    <URLs>
+        <URL> https://raw.githubusercontent.com/that-one-guy-alive/Epic-Mickey-Load-Remover/refs/heads/main/rebrushedremover.asl</URL>
+    </URLs?
+    <Type>Script</Type>
+    <Description> Load Removal Made Possible By NolifeVV</Description>
+    </AutoSplitter>
+</AutoSplitters>
```

Comment:
There shouldn't be more than 1 closing AutoSplitters tag. See how other PRs do it